modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -163,6 +163,17 @@ For request headers, these adjustments are applied only for routes that have the
 * `threadCount` specifies the number of threads to create per HAProxy process. Creating more threads allows each Ingress Controller pod to handle more connections, at the cost of more system resources being used. HAProxy
 supports up to `64` threads. If this field is empty, the Ingress Controller uses the default value of `4` threads. The default value can change in future releases. Setting this field is not recommended because increasing the number of HAProxy threads allows Ingress Controller pods to use more CPU time under load, and prevent other pods from receiving the CPU resources they need to perform. Reducing the number of threads can cause the Ingress Controller to perform poorly.
 
+* `clientTimeout` specifies how long a connection will be held open while waiting for a client response. If unset, the default timeout is 30s.
```

Comment:
> Use past or future tense only when you cannot use present tense or it does not make sense to use present tense.

If 30s is meant to be a specific value, you need to enclose it in backticks. Otherwise, you need a space: 30 s.

(These apply to all of these list items.)

Reply:
For each of the default timeouts, the unit is a part of the value. They'd be specified like this in yaml:
```
clientTimeout: "30s"
```
So backticks seems like the right choice to me